nyaa/routes.py

```diff
@@ -6,7 +6,8 @@
 from nyaa import torrents
 from nyaa import backend
 from nyaa import api_handler
-from nyaa.search import search_elastic, search_db
+from nyaa import api_torznab
+from nyaa.search import search
```

Comment:
`search_db` is still used at [L311](https://github.com/Taloth/nyaa/blob/f569e8149db24118dda9a76e4c770ac093b01cf5/nyaa/routes.py#L311) but is no longer imported